rosdep/python.yaml

```
@@ -6311,6 +6311,26 @@ python3-helyos_agent_sdk-pip:
   '*':
     pip:
       packages: [helyos_agent_sdk]
+python3-hid:
+  debian:
+    pip:
+      packages: [hid]
+  fedora:
+    pip:
+      packages: [hid]
+  ubuntu:
+    pip:
+      packages: [hid]
+python3-hidapi:
+  debian: [python3-hid]
+  fedora: [python3-hidapi]
+  ubuntu: [python3-hid]
```

In RHEL 9, this package is provided by EPEL: https://packages.fedoraproject.org/pkgs/python-hidapi/python3-hidapi/

```suggestion
  rhel:
    '*': [python3-hidapi]
    '8': null
  ubuntu: [python3-hid]
```